### DISCUSSION TITLE
Workers are registering but not visible

### DISCUSSION THREAD

**Discussion by hc2p** [concourse/concourse#7496] [Help & Support]
We are running two workers within Docker-for-Mac on a MacPro. They are supposed to connect to the remote TSA on our Kubernetes cluster. 
It works fine when starting native workers, I see them and two other workers we have running on the cluster:

```
$ fly workers 
concourse-worker-0  18          linux     none  none  running  2.3      52m40s
concourse-worker-1  21          linux     none  none  running  2.3      53m26s
macpro-worker-01    0           darwin    none  none  running  2.3      9d
macpro-worker-02    0           darwin    none  none  running  2.3      1h29m
```

There are no errors coming from the workers running as Docker containers, even success messages:
`worker-02_1  | {"timestamp":"2021-09-02T10:46:04.232443257Z","level":"info","source":"worker","message":"worker.beacon-runner.beacon.registered","data":{"session":"4.1"}}`

Any idea what I can do?

**Comment by taylorsilva** [comment]
Haven't seen this before. In the web nodes do you see a log line with `tsa.connection.channel.command.register.start` or `tsa.connection.channel.command.register.done`?

If the check the `workers` table in the db, is the worker listed there?

**Reply by hc2p** [reply]
Ah! I pointed the worker to the wrong TSA domain (while testing things). Sorry for the confusion. Everything works well now. Thanks!
